css-layouts/responsive-layouts/src/exercises/exercise3/index.js

```diff
@@ -0,0 +1,89 @@
+/*
+  ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
+  
+  💡 For this final exercise, you will need to combine what
+  💡 you learned from the first two exercises: the size prop,
+  💡 and all of the various flex props to create a fully responsive
+  💡 layout.
+
+  💡 Below, you'll see a bunch of component imports that
+  💡 are named after page elements you may see on a website
+  💡 or application. These are what you'll be wrapping in 
+  💡 `Block` components to replicate the layout you see
+  💡 on the right side of the browser window.
+
+  ⚠️ Much like the sizes prop, the flex props also accept
+  ⚠️ an object of values. For example: 
+
+  ```
+    flexDirection={{
+      small: "row",
+      large: "column"
+    }}
+  ```
+
+  💡 Would define the flex axis on the container to be vertical
+  💡 on small screens, and then horizontal on large screens.
+
```

Comment:
It would be helpful to comment on the specific flex property that gets applied within the object, in addition to saying it's vertical/horizontal here. That is to say, specifying that the flex property "row" is applied on small devices, and "column" on large devices.